### DISCUSSION TITLE
Presetting current time in MediaPlayer isnt allowing me to capture the currentTime prop

### DISCUSSION THREAD

**Discussion by SaaSManDan** [vidstack/player#1161] [General]
When I preset the current time in media player for a certain time, i can no longer console log the currentTime when I press pause. 

```tsx
function handleOnPlay(){
        console.log("Began playing at: " + currentTime)
       
    }

    function handleOnPause(){
        console.log("Paused at " + currentTime)
    }

  <MediaPlayer 
    title="Sprite Fight" 
    src={videoUrl}
    ref={mediaPlayerRef}
    onPlay={handleOnPlay}
    onPause={handleOnPause}
   currentTime={10}
  >
    <MediaProvider />
    <DefaultVideoLayout
        icons={defaultLayoutIcons}
    />
</MediaPlayer>
```

**Comment by mihar-22** [comment]
I don't understand the problem, please provide a [repro](https://github.com/vidstack/player/blob/main/.github/REPRO.md). What do you mean by, "I can't `console.log` the current time when you press pause?" I just tested it and it works fine.

**Reply by SaaSManDan** [reply]
<img width="270" alt="Screenshot 2024-02-13 at 10 01 07 AM" src="https://github.com/vidstack/player/assets/72174880/e9ed6e6d-3fcc-40c2-90b7-58313550820d">
<img width="220" alt="Screenshot 2024-02-13 at 10 01 15 AM" src="https://github.com/vidstack/player/assets/72174880/63a791b8-7c30-4815-bab1-d90f528da97b">

What I mean is that I paused at 13 seconds, however, it is console logging 0 when it should be console logging 13 seconds

**Reply by mihar-22** [reply]
How is `currentTime` being updated?

**Reply by SaaSManDan** [reply]
I'm not updating it any way

**Reply by SaaSManDan** [reply]
Here's the repro -- https://stackblitz.com/edit/stackblitz-starters-mbdxnv?file=app%2Fpage.jsx Interestingly enough, it's working fine in this environment. Perhaps it could be my browser? I was originally using Firefox Private browsing but I made this Stackblitz repro on Chrome

**Reply by SaaSManDan** [reply]
Just tested that theory and I'm still having an issue in my own environment in Chrome :/

**Reply by SaaSManDan** [reply]
I moved the video player into its own component and it seems to be working now. Perhaps something else was interfering with it in the parent page, haven't been able to identify it but glad it's working now
